src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{

```

Comment:
There should be a space between the class name and the curly brace to adhere to Java code conventions.